.github/workflows/docs.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+name: Sync `docs` directory to ReadMe
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+  
+jobs:
+  Update-Documentation:
+    name: Update the documentation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+       
+      - name: Update using rdme
+        uses: readmeio/rdme@7.0.0
+        with:
+          rdme: docs ./docs --key=${{ secrets.README_API }} --version=1.2
```

Comment:
`--version=1.2` as a static version to update is error prone. When pushing new releases to `master`, the version will be (should be) different. No recommended way forward yet...still looking...

Reply:
I don't disagree, but also didn't have a solution at this time.  Perhaps bumping the version here becomes part of the release process?

Reply:
What was the resolution on this?

Reply:
@kylewillmon I think we should take @maxrake advice and combine into `Release.yml`. We should be able to determine the version from the tag and can hopefully provide that in the `rdme` command. I don't have time to do this immediately, so I will create a ticket for the follow-on work.

Reply:
Decided to use `v1-latest` as the version of the docs that align with `master` and `v1-development` as the version that aligns to the `development` branch. This should work until we decide to support older versions of the user docs.